<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDartboard</class>
 <widget class="QDialog" name="DialogDartboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Dartboard" name="DartboardView">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-40</y>
     <width>751</width>
     <height>771</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../rsrc/resources.qrc">:/img/Dartboard.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>80</y>
     <width>201</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Wurf 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="wurf1">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Wurf 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="wurf2">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Wurf 3:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="wurf3">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Koordinaten">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>X: = 0 , Y:= 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="naechsterSpieler">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>450</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>NÃ¤chster Spieler</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undoLetzterWurf">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>530</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>UNDO letzter Wurf</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Dartboard</class>
   <extends>QLabel</extends>
   <header location="global">View/dartboard.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../rsrc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
